### ISSUE TITLE
[Bug]: AG-Grid Accessibility Issue -1 )aria-hidden-focus 2)scrollable-region-focusable

### ISSUE THREAD

**Issue by PrajaktaBlog** [ceolter/ag-grid#11120]
### Link to reproducible scenario

https://codesandbox.io/p/sandbox/stoic-darkness-3d89zr?file=%2Fsrc%2FApp.tsx%3A65%2C27

### Describe the bug

Steps to reproduce 
Providing a Reproducible Scenario
I have attached a codesandbox link below to reproduce the issue
https://codesandbox.io/p/sandbox/stoic-darkness-3d89zr?file=%2Fsrc%2FApp.tsx%3A65%2C27

Steps to reproduce the issue

Open the codesandbox link, run the application
Enable fast pass application to run the accessibility test
Current behavior
Ag-grid is throwing an accessibility issue stating that an element should be focusable.

![Image](https://github.com/user-attachments/assets/91a0b863-f4b0-4785-8372-45651074f84a)

Expected behaviour

![Image](https://github.com/user-attachments/assets/2b9b16a0-2f1c-493f-974f-6948b3023e6c)

"ag-grid-react": "32.0.2",
  "react": "^18.2.0",

### Version

18.2.0

### Does the issue occur for a specific framework only?

All languages

### Is the issue only observable on a specific browser?

All browsers

**Comment by ava-utting** [comment]
Hi @PrajaktaBlog ,

Thank you for reaching out.

We’ve tested this in the latest version of AG Grid and can confirm the issue can no longer be reproduced. In light of this, we’ll go ahead and close the issue.

Thank you again for raising this with us.

Kind Regards,
Ava
